actors/builtin/miner/miner_test.go

```diff
@@ -781,6 +835,76 @@ func (h *actorHarness) submitWindowPost(rt *mock.Runtime, deadline *miner.Deadli
 	rt.Verify()
 }
 
+func (h *actorHarness) declareFaults(rt *mock.Runtime, expectedReward abi.TokenAmount, totalQAPower abi.StoragePower, fee abi.TokenAmount, faultSectorInfos ...*miner.SectorOnChainInfo) {
+	rt.SetCaller(h.worker, builtin.AccountActorCodeID)
+	rt.ExpectValidateCallerAddr(h.worker)
+
+	ss, err := faultSectorInfos[0].Info.SealProof.SectorSize()
```

Comment:
This is needed frequently enough that we should just put the miner-constant proof type and sector size into the harness object to save the verbosity of all these lookups